minerl/env/_multiagent.py

```diff
@@ -481,6 +513,14 @@ def reset(self) -> Any:
             # the episode in a cascading fashion
             self._seed = None
 
+    def _reset_video_recorders(self, obs, agent_ind):
+        # TODO figure out what other metadata we might want saved?
+        video_path = self.video_directory / f"{agent_ind}_{self._seed}_{round(time.time())}.mp4"
+        if self.video_writers[agent_ind].is_open():
+            self.video_writers[agent_ind].close()
+        self.video_writers[agent_ind].open(video_path)
```

Comment:
It's hard to check via github but does this handle the case where I ask to record agents 3 and 4? Do we only call _reset_video_recorders for each self.record_agents?

Reply:
Yes, this is only called for agents that are in self.record_agents (indeces of agents to-be-recorded).